includes/DummyData/Importer.php

```diff
@@ -0,0 +1,290 @@
+                    array(
+                        'sku'  => $sku,
+                        'item' => $parsed_data,
+                    )
+                );
+                continue;
+            }
+
+            if ( $sku_exists && ! $update_existing ) {
+                $data['skipped'][] = new WP_Error(
+                    'woocommerce_product_importer_error',
+                    esc_html__( 'A product with this SKU already exists.', 'dokan-lite' ),
+                    array(
+                        'sku'  => esc_attr( $sku ),
+                        'item' => $parsed_data,
+                    )
+                );
+                continue;
+            }
+
+            if ( $update_existing && isset( $parsed_data['sku'] ) && ! $sku_exists ) {
+                $data['skipped'][] = new WP_Error(
+                    'woocommerce_product_importer_error',
+                    esc_html__( 'No matching product exists to update.', 'dokan-lite' ),
+                    array(
+                        'sku'  => esc_attr( $sku ),
+                        'item' => $parsed_data,
+                    )
+                );
+                continue;
+            }
+
+            $result = $this->process_item( $parsed_data );
+
+            if ( is_wp_error( $result ) ) {
+                $data['failed'][] = [
+                    'error_data' => $result->get_error_message(),
+                    'item'       => $parsed_data,
+                ];
+            } elseif ( $result['updated'] ) {
+                $data['updated'][] = $result['id'];
+            } else {
+                $data['imported'][] = $result['id'];
+
+                add_post_meta( $result['id'], 'dokan_dummy_data', true, true );
+                wp_update_post(
+                    array(
+                        'ID'          => $result['id'],
+                        'post_author' => $this->vendor_id,
+                    )
+                );
+            }
+
+            $index ++;
+        }
+
+        return $data;
+    }
+
+    /**
+     * Check if the product is from my store
+     *
+     * @param int|WC_Product $product
+     *
+     * @return bool
+     */
+    private function is_my_product( $product ) {
+        return $this->vendor_id === dokan_get_vendor_by_product( $product )->get_id();
+    }
+
+    /**
+     * Remove all dummy data ( products and vendors ) that has 'dokan_dummy_data' meta key
+     *
+     * @since DOKAN_SINCE
+     *
+     * @return string
+     */
+    public function clear_all_dummy_data() {
+        $all_products = get_posts(
+            array(
+                'post_type'   => 'product',
+                'numberposts' => -1,
+                'meta_key'    => 'dokan_dummy_data', // phpcs:ignore
+                'post_status' => 'any',
+            )
+        );
+        foreach ( $all_products as $product ) {
+            wp_delete_post( $product->ID, true );
+        }
+
+        $all_vendors = dokan()->vendor->get_vendors(
```

Comment:
I think we need to delete orders also created for these vendors, otherwise, this might create some unexpected problems eg: commissions being calculated as zero.